client/elf.js

```diff
@@ -0,0 +1,272 @@
+const logs = {}
```

Comment:
is this anywhere on git that could be referred to in the package.json ?

https://github.com/tylerchilds/wallet-attached-storage-server/pull/2